.vscode/launch.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
Hi @zshall thank you for your PR.
Can you please remove this file, as it is vscode configuration related to your local dev environment.

Reply:
Yup, removed.